renderdoc/driver/vulkan/vk_stringise.cpp

```diff
@@ -4320,3 +4320,52 @@ rdcstr DoStringise(const VkGeometryTypeKHR &el)
   }
   END_ENUM_STRINGISE();
 }
+
+template <>
+rdcstr DoStringise(const VkImageCompressionFixedRateFlagBitsEXT &el)
+{
+  BEGIN_ENUM_STRINGISE(VkImageCompressionFixedRateFlagBitsEXT);
```

Comment:
Since this is a bitfield it should use the bitfield macros (`BEGIN_BITFIELD_STRINGISE` / `END_BITFIELD_STRINGISE` + `STRINGISE_BITFIELD_BIT` + `STRINGISE_BITFIELD_VALUE`) - otherwise this will not handle multiple bits being set.

Also we don't stringise MAX_ENUM typically as it's not a valid value.